test/redeyed-smoke.js

```diff
@@ -21,7 +21,7 @@ test('tap', function (t) {
     , 'resolve/test/precedence/'
     , 'is-relative/index.js'
     , 'is-buffer/test/'
-    , 'lodash/utility/'
+    , 'lodash/'
```

Comment:
The regex is just inviting another cat-and-mouse the game. The real problem is that we have to second guess every CI run because the list of modules to be checked is outside our control (transitive dependencies).

There is no need to eliminate the smoke test. We just have to find a way to reliably include source file which has `var` keyword and exclude the one without. Here is what I have in mind:
```js
hasVar = esprima.tokenize(code).some(function (t) {
  return t.type === 'Keyword' && t.value === 'var'
})
```
(Possible drawback: Esprima dependency)